payment_system/models.py

```diff
@@ -1,45 +1,77 @@
 from django.conf import settings
+from django.contrib.auth import get_user_model
 from django.db import models
+from django.utils.timezone import now
+
 from data_ocean.models import DataOceanModel
 
 
+class ProjectSubscription(DataOceanModel):
+    STATUSES = [
+        ('active', 'active'),
+        ('past', 'past'),
+        ('future', 'future'),
+    ]
+    project = models.ForeignKey('Project', on_delete=models.CASCADE)
+    subscription = models.ForeignKey('Subscription', on_delete=models.CASCADE)
+    status = models.CharField(choices=STATUSES, max_length=10)
+    expiring_date = models.DateField(null=True)
+
+    class Meta:
+        verbose_name = "relation between the project and its subscriptions"
+
+
 class Project(DataOceanModel):
     name = models.CharField(max_length=50)
-    description = models.CharField(max_length=500, null=True)
-    token = models.CharField(max_length=40, null=True)
+    description = models.CharField(max_length=500, null=True, blank=True)
+    token = models.CharField(max_length=40, unique=True)
     disabled = models.BooleanField(default=False)
     users = models.ManyToManyField(settings.AUTH_USER_MODEL)
+    subscriptions = models.ManyToManyField('Subscription', through=ProjectSubscription)
+
+    def add_user_to_project(self, user_id):
+        user = get_user_model().objects.get(id=user_id)
+        self.users.add(user)
+        return self
+
+    def remove_user_from_project(self, user_id):
+        user = get_user_model().objects.get(id=user_id)
+        self.users.remove(user)
+        return self
+
+    def disable_project(self):
+        self.disabled = True
+        self.deleted_at = now()
```

Comment:
```suggestion
        self.deleted_at = timezone.now()
```
`now` is present in many libraries, need to more explicitly indicate where it is from

Reply:
Come on, in all other code we use now